README.md

```diff
@@ -3,6 +3,6 @@ A server for querying placement objects in *The Legend of Zelda: Breath of the W
 
 Run build.ts to generate a map database before starting the server for the first time.
 
-    ts-node build.ts -a ../botw/Actor
+    ts-node build.ts -a ../botw/Actor -e ../botw/EcoSystem
```

Comment:
IMO we should just ship the beco files (and put them in content/ecosystem/) -- there are only like 3-4 beco files and they're all tiny.